python/AMI_NOT_PUBLIC_CHECK/AMI_NOT_PUBLIC_CHECK.py

```diff
@@ -0,0 +1,406 @@
+#
```

Comment:
Not the latest RDK template. Please upgrade

Reply:
Upgraded [rdk 0.4.3]